### DISCUSSION TITLE
xcodebuild in post build script

### DISCUSSION THREAD

**Discussion by ubangy** [codemagic-ci-cd/codemagic-docs#1541] [Q&A]
Hi all,

We are trying to run `xcodebuild build-for-testing` in a post build script, without using codemagic.yaml. We have setup the developer portal key and the code signing identities. However we get the following error:
`Error (Xcode): No profiles for 'com.example.app' were found`
We have tried adding `xcode-project use-profiles` to the build script but this does not find any signing info or provision profiles even though they are setup.
Is there a way to do this without having to change over to codemagic.yaml?

Thanks in advance.

**Comment by icarusdust** [comment]
Hi @ubangy, if you do not see the profile set up on Xcode under the `xcode-project use-profiles` step, then first off double check if your project bundle id and the app bundle id in App Store are matching. I am assuming that you have an instance of `com.example.app` in your project as I do not think that its your actual project id. Double check these and if you need us to further investigate it, please provide a build id as well.

**Reply by ubangy** [reply]
Yes, they are the same. You can check this build for example: 642d793f1e33cc6085fceae4
Thanks heaps.

**Reply by ubangy** [reply]
@icarusdust I got it working. I grabbed the mobileprovision files from xcode and add them as env variables using base64. Thanks for your help. 

It would be nice though if you didn't have to add them as env variables. Does CodeMagic copy the provisioning profiles across for every build? Like I said above, they are in the codemagic.yaml team settings, but I am trying this from a post build script not `codemagic.yaml`
